config/security.java.policy-server

```diff
@@ -131,6 +131,7 @@ grant principal org.ow2.proactive.authentication.principals.GroupNamePrincipal "
 	//"false" means user can get full state and listen for any events.
     permission org.ow2.proactive.scheduler.permissions.HandleOnlyMyJobsPermission "true";
     permission org.ow2.proactive.scheduler.permissions.ChangePriorityPermission "1,2,3";
+    permission org.ow2.proactive.scheduler.permissions.HandleJobsWithGenericInformationPermission "";    
```

Comment:
no, it will work just fine, I wanted to leave an empty one in the USER group in any case to make it easier to use....